types.go

```diff
@@ -227,6 +227,26 @@ type Symbol struct {
 
 type SymbolCode uint64
 
+func StringToSymbolCode(str string) (SymbolCode, error) {
```

Comment:
A few tests would be needed before merging that, are ok adding them?

Reply:
I don't know how to add the test, I deployed a contract to verify it on EOS.